AMW_rest/src/main/java/ch/mobi/itc/mobiliar/rest/environments/EnvironmentsRest.java

```diff
@@ -24,10 +28,13 @@ public class EnvironmentsRest {
     @Inject
     ContextLocator contextLocator;
 
+    //TODO: EnvironmentsScreenDomainService is deprecated.
```

Comment:
remove the todo: the class is already annotated with `@deprecated` - the TODO does not help here.